src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ndelius/GetSentencesForPersonTest.kt

```diff
@@ -65,9 +66,11 @@ class GetSentencesForPersonTest(
           listOf(
             generateTestSentence(
               dateOfSentencing = LocalDate.parse("2009-07-07"),
+              isActive = false,
             ),
             generateTestSentence(
               dateOfSentencing = LocalDate.parse("2009-09-01"),
+              isActive = false,
```

Comment:
Is there something significant about the date being different whilst isActive remains the same? Otherwise should this be again testing all of the possible values (true, false, null)?